### ISSUE TITLE
avoid Spam messages

### ISSUE THREAD

**Issue by xAstroBoy** [lossless1024/StreaMonitor#52]
Console keeps spamming a streamer status, when something happens, instead of repeating it on loop, maybe say it once, then if the status changes, update it without spamming it.

**Comment by lossless1024** [comment]
Good idea. The logging also needs some rework. I put it on my to-do list.

**Comment by xAstroBoy** [comment]
No news about this?

**Comment by xAstroBoy** [comment]
check #114  for a simple fix.
